packages/metrics/src/config/routes.ts

```diff
@@ -512,6 +516,24 @@ export const getRoutes = () => {
         tags: ['api']
       }
     },
+    {
+      method: 'GET',
+      path: '/vsExport',
+      handler: vsExportHandler,
+      config: {
+        tags: ['api'],
+        auth: false
+      }
+    },
+    {
+      method: 'GET',
+      path: '/fetchVSExport',
```

Comment:
@sadmananik ```auth: false``` should be removed for this handler I think

Reply:
Yes. Since we call this from the client side so this should be authenticated.